packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectIModel.tsx

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import "./SelectIModel.scss";
+
+import { IModelGrid, IModelGridProps } from "@itwin/imodel-browser-react";
+import { useNavigate } from "@reach/router";
+import React from "react";
+
+export const SelectIModel = ({ accessToken, projectId }: IModelGridProps) => {
```

Comment:
when im on this pg theres no way for me to navigate back, probably cause the menu is missing on my end, but wondering if we need a header

Reply:
the goal was to use menus for this (i.e. opening a new file/workspace in VS Code). It's controversial I know. I'd like to get others thoughts once I get the menus working on Windows. We could move to a sidebar, but I think its unnecessary real estate in a desktop app where menus are standard.

Reply:
Headers should be working now on Windows. @calebmshafer This was due to ElectronHost attempting to force auto-hiding the menubar for some reason. I was able to workaround this.

Ultimately, if we are good with this UX, I think the real solution is a frameless window with a custom menu/title bar like VS Code (https://github.com/AlexTorresSk/custom-electron-titlebar). There is some effort there though that I think can wait until another minor version.